app/uk/gov/hmrc/tariffclassificationfrontend/models/Case.scala

```diff
@@ -18,11 +18,6 @@ package uk.gov.hmrc.tariffclassificationfrontend.models
 
 import java.time.{Clock, Duration, ZonedDateTime}
 
-case class CaseStatus
```

Comment:
You introduced an enum to replace this String, but in Case and CaseStatusChange you are still using the String type for the status

Reply:
Fixed Case